packages/ui-range-input/src/RangeInput/index.tsx

```diff
@@ -174,18 +175,14 @@ class RangeInput extends Component<RangeInputProps, RangeInputState> {
           false,
           'RangeInput should have a `value` or `defaultValue` set for the value to be displayed. If no value has to be displayed, set `displayValue={false}`.'
         )
-
         return null
       }
-
+      const props = { [RangeInput.outputLocatorAttribute]: this.id }
       return (
         <ContextView background="inverse" placement="end center">
-          <output
-            htmlFor={this.id}
-            css={this.props.styles?.rangeInputInputValue}
-          >
+          <div {...props} css={this.props.styles?.rangeInputInputValue}>
```

Comment:
`output` was misused here, its meant to be used as a [result of a calculation not to display a value](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/output). It caused the value to be read twice in VO and NVDA